protocol/communities/persistence.go

```diff
@@ -1221,7 +1221,7 @@ func (p *Persistence) GetCommunityChatIDs(communityID types.HexBytes) ([]string,
 func (p *Persistence) GetAllCommunityTokens() ([]*token.CommunityToken, error) {
```

Comment:
It would be nice to refactor a common part for `GetAllCommunityTokens` and `GetCommunityTokens`